docs/DeveloperGuide.md

```diff
@@ -502,148 +502,174 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use Cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
-
-**Use case: Delete a company**
+<div markdown="block" class="alert alert-info">
+**:information_source: Note:**<br>
+For all use cases below, the **System** is `LinkMeIn` and the **Actor** is the `user`, unless specified otherwise.
+</div>
 
-**MSS**
 
-1.  User requests to list companies
-2.  AddressBook shows a list of companies
-3.  User requests to delete a specific company in the list
-4.  AddressBook deletes the company
+**Use Case: UC01 - List all Companies**
 
-    Use case ends.
+**MSS** <br>
+1. User requests to list all companies. 
+2. LinkMeIn displays the full list of companies. <br>
+Use case ends.
 
-**Extensions**
+**Extensions** <br>
+* 1a. LinkMeIn detects an invalid command format error in the input.
+  * 1a1. LinkMeIn displays an error message. 
+  * 1a2. User enters a new command. <br>
```

Comment:
```suggestion
  * 1a2. User enters a new command to list all companies. <br>
```
Not sure if its ok to be vague. If the user enters some other command then step 2 does not make sense

Reply:
okie will edit this!